service/src/main/java/com/softserve/borda/services/impl/UserServiceImpl.java

```diff
@@ -106,7 +102,7 @@ public User deleteCommentFromUser(Long userId, Comment comment) {
     }
 
     @Override
-    public List<Comment> getAllCommentsByUserId(Long userId) {
-        return getUserById(userId).getComments();
+    public List<Comment> getAllCommentsByUser(User user) {
+        return user.getComments();
```

Comment:
Do not fetch entity-related entities using getters, use the service layer instead.

Reply:
Fixed in ReworkService branch/ticket